src/providers/formattingProvider.ts

```diff
@@ -90,7 +90,7 @@ export class FormatProvider implements vscode.DocumentFormattingEditProvider {
 
         for (let lineIndex = 0; lineIndex < document.lineCount; lineIndex++) {
             const originalLine = document.lineAt(lineIndex).text;
-            const purifiedLine = CodeUtil.purify(originalLine.toLowerCase());
+            const purifiedLine = CodeUtil.purify(originalLine).toLowerCase();
```

Comment:
Are the regex's coming into this `CodeUtil.purify` expected to be lowercase? If not, we've made a big change here that might not work as expected.

Reply:
Asking because I know another #221 has some major changes here as well

Reply:
RegExp inside `CodeUtil.purify` is case-insensitive, string with any case acceptable.

Line 217: `let regExp = new RegExp(pattern, 'i');`